### PR TITLE
Update Picture-in-Picture API compat data

### PR DESCRIPTION
**Pull request by foolip**
These changes were prepared by @gsouquet in another PR:
https://github.com/mdn/browser-compat-data/pull/7028

Closes https://github.com/mdn/browser-compat-data/pull/7028.

Co-authored-by: Germain <germain@grabyo.com>
Co-authored-by: Daniel D. Beck <daniel@ddbeck.com>

### CODE REVIEW

**Comment by foolip** [comment]
The initial state of this PR is https://github.com/mdn/browser-compat-data/pull/7028 without modifications. I'll now make the changes needed.

File: api/HTMLVideoElement.json [hunk 1]
```diff
@@ -49,15 +49,39 @@
       },
       "autoPictureInPicture": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/autoPictureInPicture",
```
Comment (foolip) [inline api/HTMLVideoElement.json:52]: This does not seem to be supported yet:
https://mdn-bcd-collector.appspot.com/tests/api/HTMLVideoElement/autoPictureInPicture
https://www.chromestatus.com/feature/5317876315586560

**Comment by foolip** [comment]
This is quite complicated, but ready for review now. Main things to know:
- `autoPictureInPicture` is a separate feature, under a separate flag in Chromium. Only shipped in Safari so far.
- `PictureInPictureEvent` was initially called `EnterPictureInPictureEvent` in both Chrome and Safari. The "leavepictureinpictureevent" existed, but fired a regular `Event`. That was then changed to fire `PictureInPictureEvent`, so that those events would also get the `pictureInPictureWindow` attribute.

**Review by ddbeck — APPROVED** [review]
Thank you, @foolip (and @gsouquet)! 🎉

**Pull request merged**